src/entity/posts/Post.ts

```diff
@@ -288,4 +288,8 @@ export class Post {
     },
   )
   public codeSnippets: Promise<PostCodeSnippet[]>;
+
+  @Column({ type: 'integer', default: 0 })
+  @Index('IDX_post_awards')
+  awards: number;
```

Comment:
Don't we need to store also the total amount? I don't remember what we show on the frontend

Reply:
If this is what you meant? https://dailydotdev.slack.com/archives/C07VA1FJTDK/p1741273583885919

Reply:
yes, perfect